### PR TITLE
The bug fix for a printout of a out of boundary array.

### PR DESCRIPTION
**Pull request by WenMeng-NOAA**
A bug fix for a printout of an array with out of boundary from NCO's feedback.

### CODE REVIEW

File: sorc/ncep_post.fd/INITPOST_GFS_NEMS_MPIIO.f [hunk 1]
```diff
@@ -1111,7 +1111,7 @@ SUBROUTINE INITPOST_GFS_NEMS_MPIIO(iostatusAER)
             enddo
           enddo
           if (me == 0) print*,'sample model pint,pmid' ,ii,jj,l &
```
Comment (XianwuXue-NOAA) [inline sorc/ncep_post.fd/INITPOST_GFS_NEMS_MPIIO.f]: Why only delete ",pmid(ii,jj,l)"? however, the message still has pmid.

Comment (WenMeng-NOAA) [reply sorc/ncep_post.fd/INITPOST_GFS_NEMS_MPIIO.f]: Good catching. pmid is remove from that printout line.

**Comment by XianwuXue-NOAA** [comment]
Thanks. I am still learning why just deleting it.

**Comment by WenMeng-NOAA** [comment]
pmid has the dimensions imXjmX64 while pint has the dimension imXjmX65. The printout line try to print out pmid(i,j,65). This is a bug only found at runtime with turning on debug mode.

**Comment by XianwuXue-NOAA** [comment]
I see lp1 is for the upper bound for pint instead of pmid. Therefore, there is no need pmid here.

**Review by XianwuXue-NOAA — APPROVED** [review]

**Pull request merged**